Mathlib/Algebra/Star/NonUnitalSubalgebra.lean

```diff
@@ -0,0 +1,1224 @@
+/-
+Copyright (c) 2018 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau, Yury Kudryashov
+-/
+import Mathlib.Algebra.Algebra.NonUnitalSubalgebra
+import Mathlib.Algebra.Star.StarAlgHom
+import Mathlib.Algebra.Star.Center
+
+/-!
+# Non-unital Star Subalgebras
+
+In this file we define `NonUnitalStarSubalgebra`s and the usual operations on them
+(`map`, `comap`).
+
+## TODO
+
+* once we have scalar actions by semigroups (as opposed to monoids), implement the action of a
+  non-unital subalgebra on the larger algebra.
+-/
+
+#align non_unital_subalgebra NonUnitalSubalgebra
```

Comment:
Yes, sorry. They are mathport legacy. It's handy to keep them around as long as possible though, especially if you have more ported files downstream (which I did) and also so that you can run `scripts/fix-comments` to deal with docstrings.